lib/MusicBrainz/Server/Controller/Role/EditRelationships.pm

```diff
@@ -191,8 +191,8 @@ role {
 
                 $args{entity0} = $field->{forward} ? $source : $target;
                 $args{entity1} = $field->{forward} ? $target : $source;
-                $args{entity0_credit} = $field->{entity0_credit};
-                $args{entity1_credit} = $field->{entity1_credit};
+                $args{entity0_credit} = $field->{entity0_credit} if exists $field->{entity0_credit};
```

Comment:
And you use `exists` here because you need to check if this key is actually in a hash? Compared to `defined` below, where you actually know that key is there and want to see if value is actually *defined*?